docusaurus/docs-internals/editor-controls/ui-controls/button.md

```diff
@@ -0,0 +1,52 @@
+|       `devices`       |    string    | `"all"` | Define the devices where the control will be rendered. Options: `"all"`, `"desktop"`, `"responsive"`. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+|      `disabled`       |   boolean    |  false  | Determines whether the control is disabled or enabled.                                                                                                                                                                                                                           |
+|       `onClick`       | VoidFunction |    -    | Function to be called when the button is clicked.                                                                                                                                                                                                                                |
+|     `config.icon`     |    string    |    -    | The icon of the button.                                                                                                                                                                                                                                                          |
+|     `config.text`     |    string    |    -    | The text displayed on the button.                                                                                                                                                                                                                                                |
+|   `config.reverse`    |   boolean    |    -    | Determines whether the button layout is reversed.                                                                                                                                                                                                                                |
+| `config.reverseTheme` |   boolean    |    -    | Configures whether the theme is reversed for the button.                                                                                                                                                                                                                         |
+|    `config.title`     |    string    |    -    | The title of the button.                                                                                                                                                                                                                                                         |
+|    `config.value`     |   boolean    |    -    | The value of the button.                                                                                                                                                                                                                                                         |
```

Comment:
value is not used 
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/e25f2138-72b9-404a-a51f-df4d004eaea7)

and by onClick we don't send anythink, so i think we can delete `value` from this table